imod_coupler/drivers/dfm_metamod/mf6_wrapper.py

```diff
@@ -122,16 +122,55 @@ def get_river_stages(
         stage = bound[:, 0]
         return stage
 
-    def get_river_flux(
+    def set_correction_flux(
```

Comment:
from a modflow perspective, this function sets the pumping rate of the wells in a well package. So I think the function should be called  "set_pumping_rate" , not " set_correction_flux" d

Reply:
indeed